poolapi/apijson.go

```diff
@@ -13,31 +13,32 @@ type Response struct {
 // TODO: make JSON tags lower-case and add "_" between words
 
 type PurchaseInfo struct {
-	PoolAddress   string `json:"PoolAddress"`
-	PoolFees      string `json:"PoolFees"`
-	Script        string `json:"Script"`
-	TicketAddress string `json:"TicketAddress"`
+	PoolAddress   string  `json:"PoolAddress"`
+	PoolFees      float64 `json:"PoolFees"`
```

Comment:
Can this be changed to be a value between 0 and 1?  Easier to work with percentages this way.  I'd argue that percent formatting should only be done for display.